storybook/qmlTests/tests/tst_SwapModal.qml

```diff
@@ -599,6 +635,497 @@ Item {
+            root.swapFormData.selectedNetworkChainId = root.swapAdaptor.filteredFlatNetworksModel.get(0).chainId
+            root.swapFormData.fromTokensKey = "ETH"
+            root.swapFormData.fromTokenAmount = valueToExchangeString
+
+            let expectedToken =  SQUtils.ModelUtils.getByKey(root.swapAdaptor.processedAssetsModel, "tokensKey", "ETH")
+
+            // Launch popup
+            launchAndVerfyModal()
+
+            const payPanel = findChild(controlUnderTest, "payPanel")
+            verify(!!payPanel)
+            const amountToSendInput = findChild(payPanel, "amountToSendInput")
+            verify(!!amountToSendInput)
+            const bottomItemText = findChild(payPanel, "bottomItemText")
+            verify(!!bottomItemText)
+            const holdingSelector = findChild(payPanel, "holdingSelector")
+            verify(!!holdingSelector)
+            const maxTagButton = findChild(payPanel, "maxTagButton")
+            verify(!!maxTagButton)
+            const holdingSelectorsContentItemText = findChild(payPanel, "holdingSelectorsContentItemText")
+            verify(!!holdingSelectorsContentItemText)
+            const holdingSelectorsTokenIcon = findChild(payPanel, "holdingSelectorsTokenIcon")
+            verify(!!holdingSelectorsTokenIcon)
+
+            waitForRendering(payPanel)
+
+            compare(amountToSendInput.caption, qsTr("Pay"))
+            verify(amountToSendInput.interactive)
+            compare(amountToSendInput.input.text, valueToExchangeString)
+            compare(amountToSendInput.input.placeholderText, LocaleUtils.numberToLocaleString(0))
+            verify(amountToSendInput.input.input.edit.cursorVisible)
+            compare(bottomItemText.text, root.swapAdaptor.currencyStore.formatCurrencyAmount(0 * expectedToken.marketDetails.currencyPrice.amount, root.swapAdaptor.currencyStore.currentCurrency))
+            compare(holdingSelector.selectedItem, expectedToken)
+            compare(holdingSelectorsContentItemText.text, expectedToken.symbol)
+            compare(holdingSelectorsTokenIcon.image.source, Constants.tokenIcon(expectedToken.symbol))
+            verify(holdingSelectorsTokenIcon.visible)
+            verify(maxTagButton.visible)
+            compare(maxTagButton.text, qsTr("Max. %1").arg(root.swapAdaptor.currencyStore.formatCurrencyAmount(Math.trunc(WalletUtils.calculateMaxSafeSendAmount(expectedToken.currentBalance, expectedToken.symbol)*100)/100, expectedToken.symbol, {noSymbol: true})))
+            compare(payPanel.selectedHoldingId, expectedToken.symbol)
+            compare(payPanel.cryptoValue, 0)
+            compare(payPanel.cryptoValueRaw, SQUtils.AmountsArithmetic.fromNumber("0", expectedToken.decimals).toString())
+            verify(!payPanel.cryptoValueValid)
+
+            closeAndVerfyModal()
+        }
+
+        function test_modal_pay_input_switching_networks() {
```

Comment:
on switching test networks the form should be reset!! needs to be changed to that behaviour https://www.figma.com/design/TS0eQX9dAZXqZtELiwKIoK/Swap---Milestone-1?node-id=3410-271335&m=dev